snekbox/api/resources/eval.py

```diff
@@ -38,6 +44,8 @@ def on_post(self, req: falcon.Request, resp: falcon.Response) -> None:
         """
         Evaluate Python code and return stdout, stderr, and the return code.
 
+        The optional `args` parameter can be passed, and it would replace the "-c" option.
```

Comment:
This should be clearer

```suggestion
        A list of arguments for the Python subprocess can be specified as `args`.
        Otherwise, the default argument "-c" is used to execute the input code.
        The input code is always passed as the last argument to Python.
```